### PR TITLE
Expand the Gallantries project description

### PR DESCRIPTION
**Pull request by bebatut**
@hexylena @shiltemann @fpsom @abretaud what do you think about these changes?

### CODE REVIEW

**Review by hexylena — APPROVED** [review]

**Comment by fpsom** [comment]
LGTM! Thanks @bebatut

File: projects/10/README.md [hunk 1]
```diff
@@ -29,13 +35,20 @@ Bérénice Batut <berenice.batut@gmail.com>
 
 ## Expected outcomes
 
-- Day 1: Review existing ELIXIR TrP and GTN TtT material
-- Day 2-4: Define a learning plan and rough draft of the 3 modules
-- Day 4: Implement a template repository that can be readily re-used to easily create a course website for organizers of Galaxy-based training events (based on the GTN Smorgasbord event)
+- Training modules
+  - Review of existing materials   
+  - Learning plans for the 3 modules
+  - Rought drafts for content
```
Comment (abretaud) [inline projects/10/README.md:41]:
```suggestion
  - Rough drafts for content
```

File: projects/10/README.md [hunk 2]
```diff
@@ -1,24 +1,30 @@
...
+# Project 10: Development of training modules and infrastructure for Gallantries
 
 ## Abstract
 
 The Gallantries project is a collaboration between five European universities, members of Software Carpentry, and members of the Galaxy Project. Goal of the project is to increase bioinformatics and core data analysis skills in the field of life sciences across Europe.
 A pilot effort in 2019, started during Biohackathon in 2018, developed Hybrid Training: broadcast of a single instructor to learners in distributed classrooms with on-site helpers. This significantly improved the scalability and decreased the environmental impact of having instructors travel around Europe. With COVID-19 pandemic, hybrid and/or fully virtual training events have become the norm. To support this teaching format, training materials must be adapted, and instructors need to be trained.
+
 Main focus of the BioHackathon project will be to discuss, and create draft for training modules on microbial analysis, machine-learning, and Train-the-Trainer (TtT), specifically tailored to fit a remote/hybrid training format. Specifically, the project will include the following activities:
```
Comment (abretaud) [inline projects/10/README.md:8]:
```suggestion
Main focus of the BioHackathon project will be to discuss, and create draft for training modules on microbial analysis, genome annotation, and Train-the-Trainer (TtT), specifically tailored to fit a remote/hybrid training format. Specifically, the project will include the following activities:
```

File: projects/10/README.md [hunk 3]
```diff
@@ -1,24 +1,30 @@
...
 ## Topics
 
-Galaxy
-Machine learning
-Training Platform
+- Galaxy
+- Training Platform
+- Hybrid training
+- Train the trainers
+- Machine learning
```
Comment (abretaud) [inline projects/10/README.md:23]:
```suggestion
- Genome annotation
```

File: projects/10/README.md [hunk 4]
```diff
@@ -29,13 +35,20 @@ Bérénice Batut <berenice.batut@gmail.com>
...
+- Training infrastructure
+  - Template repository to create Galaxy-based training event websites
+  - Website with an API to list videos supporting Galaxy-based tutorials
+  - Prototype for the course builder
+  - Documentation
 
 ## Expected audience
 
-researchers with knowledge in training development, microbial data analysis, machine learning and Train the Trainer
+Researchers with knowledge in training development, microbial data analysis, machine learning, Train the Trainer and also in training organization.
```
Comment (abretaud) [inline projects/10/README.md:50]:
```suggestion
Researchers with knowledge in training development, microbial data analysis, genome annotation, Train the Trainer and also in training organization.
```

**Review by abretaud — COMMENTED** [review]
genome annotation is the new machine learning

**Pull request merged**